README.md

```diff
@@ -22,3 +22,15 @@ Local Search found. Why is this?
 
 Add the code to run your experiments, graphs, and an explanation of what you did
 to this markdown file.
+
+
+References used:
+tsp-comparison-ClaytonBrown4741
+
+I have implemented the same random function from the above repository for testing the empirical analysis of the comparison between Held Karp and Local search algorithm I implemented. However, I have changed the way the data is logged into the console. Along wiht that I made the random generator run until the input size of 25 is reached. However I was only able to have the data up until the input size of 19. Anything more than that has been ran for about close to an hour for solving the held karp. However the local search always stayed 0 because it tries a maximum of n*n iterations and stops. I ran the code again for about an hour to see if I can get the runtime for input size of 20 but I was unsuccessful. 
```

Comment:
You need to run local search for at least an hour as well.

Reply:
I have made some changes where the tests are done separately. I made sure that I had the local search run longer. So far the data I received is included in the data.txt. Please let me know if you want me to run more tests on this. 

I also updated the readme file stating what changes are made to do the changes.

Reply:
It doesn't look like your local search experiments run for at least an hour.

Reply:
can i make the maxiterations be n * n * 1000 to make it run longer. Or do I have to find a test case that runs that long

Reply:
Up to you.